components/builder-api/src/server/resources/projects.rs

```diff
@@ -115,33 +111,37 @@ fn create_project((req, body): (HttpRequest<AppState>, Json<ProjectCreateReq>))
         Err(err) => return err.into(),
     };
 
+    let conn = match req.state().db.get_conn() {
```

Comment:
This is not your fault but I'd really like to find a way to map a `Pool` error to an HttpResponse of Internal server error since we cargo cult this around all over the place

Reply:
Sure I can take a look at doing that